tools/db_crashtest.py

```diff
@@ -704,6 +702,9 @@ def finalize_and_sanitize(src_params):
         dest_params["enable_pipelined_write"] = 0
     if dest_params.get("sst_file_manager_bytes_per_sec", 0) == 0:
         dest_params["sst_file_manager_bytes_per_truncate"] = 0
+    if dest_params.get("allow_concurrent_memtable_write", 1) == 1:
+        dest_params["memtablerep"] = "skip_list"
+        dest_params["inplace_update_support"] = 0
```

Comment:
Need to move this to here so that we have 
```
if dest_params.get("unordered_write", 0) == 1:
        dest_params["txn_write_policy"] = 1
        dest_params["allow_concurrent_memtable_write"] = 1
```
above it. It's safe to do that since nothing depends on `memtablerep` and the only thing depends on `inplace_update_support` is still below it.

Reply:
There's also change in `allow_concurrent_memtable_write` in line 779.

https://github.com/facebook/rocksdb/blob/main/tools/db_crashtest.py?#L779

While line 779 change is to turn it off and this shouldn't matter, I think it's still better to set it after `allow_concurrent_memtable_write` is no longer changed by anything.

Reply:
Fixed - also moved below along for sam reason

```
    if (dest_params.get("enable_compaction_filter", 0) == 1
        or dest_params.get("inplace_update_support", 0) == 1):
        ...
```